syncstorage/tests/functional/test_storage.py

```diff
@@ -574,6 +574,63 @@ def test_delete_collection_items(self):
         res = self.app.get(self.root + '/storage/col2')
         self.assertEquals(len(res.json), 0)
 
+    def test_delete_collection_then_post(self):
+        # creating a collection of three
+        bso1 = {'id': '12', 'payload': _PLD}
+        bso2 = {'id': '13', 'payload': _PLD}
+        bso3 = {'id': '14', 'payload': _PLD}
+        bsos = [bso1, bso2, bso3]
+
+        self.app.post_json(self.root + '/storage/col2', bsos)
+        res = self.app.get(self.root + '/storage/col2')
+        self.assertEquals(len(res.json), 3)
+
+        resp = self.app.get(self.root + '/info/collections')
+        res = resp.json
+        self.assertTrue("col2" in res)
+
+        infoBefore = self.app.get(self.root + '/info/collections')
+
+        # Deleting the collection should result in:
+        #  - collection does not appear in /info/collections
+        #  - X-Last-Modified timestamp at the storage level changing
+        self.app.delete(self.root + '/storage/col2')
+        items = self.app.get(self.root + '/storage/col2').json
+        self.assertEquals(len(items), 0)
+
```

Comment:
We should probably add a ``time.sleep`` here too in order to fix the test failures.